Pod/Classes/WPMediaPickerViewController.h

```diff
@@ -234,10 +247,17 @@
 @property (nonatomic, strong, readonly, nullable) UISearchBar *searchBar;
 
 /**
- The default empty view. When `emptyViewForMediaPickerController:` is not implemented, use this property to style the mensaje.
+ The default empty view. When `emptyViewForMediaPickerController:` is not implemented,
+ use this property to style the message.
  */
 @property (nonatomic, strong, readonly, nonnull) UILabel *defaultEmptyView;
 
+/**
+ The default empty view controller. When `emptyViewControllerForMediaPickerController:` is not implemented,
+ use this property to style the message.
+ */
+@property (nonatomic, strong, readonly, nonnull) UIViewController *defaultEmptyViewController;
```

Comment:
Is it necessary to have both a `defaultEmptyView` and a `defaultEmptyViewController`?
Probably we should just have one default that will be displayed if non of the related delegates method are implemented. Does it make sense?

Not sure which would be the best option, but I'd vote for the simplest of them.

Reply:
Hey @etoledom . Good point...

So, `WPNavigationMediaPickerViewController` needs access to `defaultEmptyViewController`, so I added a method definition. I also removed `defaultEmptyViewController` as a property, and changed it to use `defaultEmptyView`.